resources/js/common/components/AchievementsListItem/AchievementDateMeta/AchievementDateMeta.tsx

```diff
@@ -70,6 +73,18 @@ export const AchievementDateMeta: FC<AchievementDateMetaProps> = ({
         </p>
       ) : null}
 
+      {isDemoted ? (
+        <span
+          className="flex cursor-help items-center"
+          title={t(
+            'This achievement cannot currently be earned because the source achievement has been demoted.',
+          )}
```

Comment:
UX: Recommend using BaseTooltip for this.

Reply:
<img width="707" height="127" alt="Image" src="https://github.com/user-attachments/assets/96289bde-80a4-4fe9-9765-20825864c673" />